addons/account/models/chart_template.py

```diff
@@ -1028,6 +1029,7 @@ class AccountReconcileModelTemplate(models.Model):
         ('percentage', 'Percentage of amount')
         ], string="Second Amount type",required=True, default='percentage')
     second_amount = fields.Float(string='Second Amount', digits=0, required=True, default=100.0, help="Fixed amount will count as a debit if it is negative, as a credit if it is positive.")
+    second_tax_include = fields.Boolean(string='Second Tax Include')
```

Comment:
use the same label as price_include field in account.tax => 'Tax Included in Price'

Reply:
is_second_tax_included